common/pom.xml

```diff
@@ -254,12 +254,6 @@
       <artifactId>tempus-fugit</artifactId>
       <scope>test</scope>
     </dependency>
-    <dependency>
-      <groupId>org.mockito</groupId>
-      <artifactId>mockito-core</artifactId>
```

Comment:
The same dependency is present at 
https://github.com/apache/hive/blob/39c88d7c1742b6ab0c87d58d333d81aefedeba3d/common/pom.xml#L362